src/main/java/com/hida/Minter.java

```diff
@@ -0,0 +1,475 @@
+
+            // if the max value of a particular digit is reached, overflow
+            if (value == tokenMaps.get(token).length()) {
+                baseMap[lastPos - k] = 0;
+                k++;
+            } else {
+                String tempId = PREFIX;
+
+                for (int i = 0; i < CHAR_MAP.length(); i++) {
+                    char character = CHAR_MAP.charAt(i);
+                    String charToken = BASE_MAP.get(character);
+                    String map = tokenMaps.get(charToken);
+
+                    tempId += map.charAt(baseMap[i]) + "";
+                }
+                System.out.println("tempid = " + tempId);
+                // add id to list
+                ID_LIST.add(tempId);
+
+                // if there wasn't over flow, increase id count
+                idsCreated++;
+
+                // reset k so it'll point to last digit again
+                k = 0;
+            }
+        }
+        return baseMap;
+    }
+
+    /**
+     * Creates a Json object based off the ids in of ID_ARRAY.
+     *
+     * @return - A reference to a String that contains Json list of ids
+     */
+    private String convertListToJson() {
+        // Jackson objects to create formatted Json string
+        String jsonString = "";
+        ObjectMapper mapper = new ObjectMapper();
+        Object formattedJson;
+        try {
+            // Object used to iterate through list of ids
+            Iterator<String> iterator = ID_LIST.iterator();
+            for (int i = 0; iterator.hasNext(); i++) {
+
+                // map desired Json format
+                String id = String.format(
+                        "{\"id\":%d,\"name\":\"%s\"}", i, iterator.next());
+
+                formattedJson
+                        = mapper.readValue(id, Object.class
+                        );
+
+                // append formatted json
+                jsonString += mapper.writerWithDefaultPrettyPrinter().
+                        writeValueAsString(formattedJson) + "\n";
+            }
+        } catch (IOException exception) {
+            System.err.println(exception.getMessage());
+        }
+
+        return jsonString;
+    }
+
+    /**
+     * Deciding whether or not to implement a check here or in Minter class
+     *
+     * @param ID - an id to check against a db
+     * @return - true if the id exists in the database
+     */
+    public boolean checkID(String ID) {
+        return this.ID_LIST.contains(ID);
+    }
+
+    /**
+     * error checking
+     */
+    public void printID_ARRAY() {
+        for (String index : this.ID_LIST) {
+            System.out.println(index);
+        }
+    }
+
+    /* typical getter and setter methods */
+    public String getCHAR_MAP() {
+        return this.CHAR_MAP;
+    }
+
+    public String getPREFIX() {
+        return this.PREFIX;
+    }
+
+    public int getLENGTH() {
+        return this.LENGTH;
+    }
+
+    public Set<String> getID_ARRAY() {
+        return ID_LIST;

```

Comment:
Along with the above comment about the style guide and constants, these should not be in caps